src/derive_utils.rs

```diff
@@ -199,3 +199,22 @@ impl<T: PyClass, I: Into<PyClassInitializer<T>>> IntoPyNewResult<T, I> for PyRes
         self
     }
 }
+
+pub trait GetPropertyValue {
+    fn get_property_value(&self, py: Python) -> PyObject;
+}
+
+impl<T> GetPropertyValue for &T
+where
+    T: IntoPy<PyObject> + Clone,
+{
+    fn get_property_value(&self, py: Python) -> PyObject {
+        (*self).clone().into_py(py)
```

Comment:
Why `Deref` here... ?

Reply:
`&self` is `&&T`,  so `self.clone()` makes `&T`. We need to deref like `(*self).clone()` to get a `T`.